tezos/new_context/src/working_tree/working_tree.rs

```diff
@@ -233,6 +236,17 @@ impl WorkingTree {
         Ok(Some(Self::new_with_tree(self.index.clone(), tree)))
     }
 
+    pub fn find_tree_ocaml(&self, key: OCaml<OCamlList<String>>) -> Result<Option<Self>, MerkleError> {
```

Comment:
@sebastiencs why are all the `_ocaml` variants needed? isn't converting the `OCamlList<String>` into a `&[&str]` to use that as keys enough?

Reply:
Maybe you didn't change this yet and what I am seeing is just the push from the rebase, if that is the case just ignore me.

Reply:
Yes it's just a push after the rebase